MycroftAI/documentation
# Swap [] and () in MD link.

#### Description
Fixes #223 

#### Type of PR
- [ ] Refactor of code (without functional changes)

#### Testing
How can someone reviewing this PR test that it is working properly?
1. Open the offending URL (see #223) in a browser
2. Replace the page URL from `linux` onward with the relative link in the source file
3. Verify that the correct URL loads: `https://mycroft-ai.gitbook.io/docs/using-mycroft-ai/customizations/wake-word#switching-wake-word-listeners`

#### Documentation
N/A
